crates/modelardb_storage/src/delta_lake.rs

```diff
@@ -253,6 +257,41 @@ impl DeltaLake {
             .map_err(|error| error.into())
     }
 
+    /// Return a [`DeltaTableWriter`] for writing to the model table with `delta_table` in the Delta
+    /// Lake, or a [`ModelarDbStorageError`] if a connection to the delta lake cannot be established
+    /// or the table does not exist.
+    pub async fn model_table_writer(&self, delta_table: DeltaTable) -> Result<DeltaTableWriter> {
+        let partition_columns = vec![FIELD_COLUMN.to_owned()];
+
+        // Specify that the file must be sorted by the tag columns and then by start_time.
+        let mut sorting_columns = Vec::new();
```

Comment:
Maybe preallocate the vec with enough capacity for the sorting columns.

Reply:
Fixed.